### PR TITLE
docs: update example in hf spaces page

### LINKED ISSUE #4567: [DOCS] Update example in HF Spaces page
## Which page or section is this issue related to?

Update the example in this [page](https://docs.argilla.io/en/latest/getting_started/installation/deployments/huggingface-spaces.html#create-your-first-dataset) to the latest `FeedbackDataset` and make it simpler.

### PR DESCRIPTION
**Pull request by sdiazlor**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #4567

**Type of change**

(Remember to title the PR according to the type of change)

- [x] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

- [x] `sphinx-autobuild` (read [Developer Documentation](https://docs.argilla.io/en/latest/community/developer_docs.html#building-the-documentation) for more details)

**Checklist**

- [ ] I added relevant documentation
- [ ] I followed the style guidelines of this project
- [ ] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the `CHANGELOG.md` file (See https://keepachangelog.com/)

### CODE REVIEW

**Review by nataliaElv — COMMENTED** [review]
It looks good to me, great work @sdiazlor !
I'll ask for @dvsrepo 's review just to have the double-check.

File: docs/_source/getting_started/installation/deployments/huggingface-spaces.md [hunk 1]
```diff
@@ -57,84 +57,66 @@ Once Argilla is running, you can use the UI with the Direct URL. This URL gives
...
+        fields={
+            "text": "I feel sad today",
+        },
+    )
+]
+dataset.add_records(records)
 ```
 
-As a next step, you can check the [Argilla Tutorials](https://docs.argilla.io/en/latest/tutorials/tutorials.html) section. All the tutorials can be run using Colab or local Jupyter Notebooks, so you can start building datasets with Argilla and Spaces!
+Congrats! You now have a dataset available from the Argilla UI to start browsing and labeling. I
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/installation/deployments/huggingface-spaces.md]: small typo here

File: docs/_source/getting_started/installation/deployments/huggingface-spaces.md [hunk 2]
```diff
@@ -57,84 +57,66 @@ Once Argilla is running, you can use the UI with the Direct URL. This URL gives
...
-You can also get the full dataset and push it to the Hub for reproducibility and versioning:
-
-```python
-# save full argilla dataset for reproducibility
-rg.load("bankingapp_sentiment").to_datasets().push_to_hub("bankingapp_sentiment")
-```
+Now, we will add the records. Create a list with the records you want to add and ensure that you match the fields with the ones specified in the previous step.
 
-Finally, this is how you can train a SetFit model using data from your Argilla Space:
+> You can also use `pandas` or `load_dataset` to [read an existing dataset and create records from it](https://docs.argilla.io/en/latest/practical_guides/create_update_dataset/records.html#add-records).
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/installation/deployments/huggingface-spaces.md]: use a local reference not to the latest direct URL

File: docs/_source/getting_started/installation/deployments/huggingface-spaces.md [hunk 3]
```diff
@@ -57,84 +57,66 @@ Once Argilla is running, you can use the UI with the Direct URL. This URL gives
...
-rg.log(rg.read_datasets(dataset, task="TextClassification"), name="bankingapp_sentiment")
 ```
 
-Congrats! You now have a dataset available from the Argilla UI to start browsing and labeling. In the code above, we've used one of the many integrations with Hugging Face libraries, which let you read hundreds of datasets available on the Hub.
-
-### Data labeling and model training
+So let's create a Dataset with two labels ("sadness" and "joy"). Don't forget to replace "your-workspace" where the dataset will be created.
 
-At this point, you can label your data directly using your Argilla Space and read the training data to train your model of choice.
+> To check your workspaces, go to "My settings" on the UI. If you need to create a new one, consult the [docs](https://docs.argilla.io/en/latest/getting_started/installation/configurations/workspace_management.html).
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/installation/deployments/huggingface-spaces.md]: same here w.r.t. URLs

**Review by davidberenstein1957 — COMMENTED** [review]
Thanks. I added some comments w.r.t. a small typo and a direct/indirect URL.

**Review by dvsrepo — APPROVED** [review]
good job, i've did some final edits as we were still referencing setfit

**Pull request merged**